src/sage/categories/filtered_modules_with_basis.py

```diff
@@ -200,8 +200,10 @@ def homogeneous_component_basis(self, d):
 
                 sage: A = GradedModulesWithBasis(ZZ).example()
                 sage: A.homogeneous_component_basis(4)
-                Lazy family (Term map from Partitions to An example of a graded module with basis:
-                             the free module on partitions over Integer Ring(i))_{i in Partitions of the integer 4}
+                Lazy family (Term map
+                 from Partitions
+                   to An example of a graded module with basis: the free module
+                      on partitions over Integer Ring(i))_{i in Partitions of the integer 4}
```

Comment:
Maybe split one more line so it gets under 80 columns?
Let me try:
```suggestion
                Lazy family (Term map
                 from Partitions
                   to An example of a graded module with basis:
                      the free module on partitions over Integer Ring(i)
                 )_{i in Partitions of the integer 4}
```
Not sure if that's actually ok... Your version is not so bad either, do as it pleases you.

Reply:
My version runs to column 91, which I think is fine because there are no # optional tags around at column 88

Reply:
Generally in reformatting doctest output, I don't introduce whitespace where no whitespace was before